sdcm/cluster.py

```diff
@@ -2149,39 +2144,53 @@ def is_scylla_installed(self, raise_if_not_installed=False):
             raise Exception(f"There is no pre-installed ScyllaDB on {self}")
         return False
 
-    def get_scylla_version(self) -> str:
-        self.scylla_version = self.scylla_version_detailed = ""
-        scylla_path = self.add_install_prefix("/usr/bin/scylla")
-        result = self.remoter.run(f"{scylla_path} --version", ignore_status=True)
+    def get_scylla_binary_version(self) -> Optional[str]:
+        result = self.remoter.run(f"{self.add_install_prefix('/usr/bin/scylla')} --version", ignore_status=True)
         if result.ok:
-            self.scylla_version_detailed = result.stdout.strip()
-            if build_id := self.get_scylla_build_id():
-                self.scylla_version = self.scylla_version_detailed
-                self.scylla_version_detailed += f" with build-id {build_id}"
-                self.log.info(f"Found ScyllaDB version with details: {self.scylla_version_detailed}")
-            self.log.debug(f"self.scylla_version_detailed={self.scylla_version_detailed}")
-        else:
-            if self.distro.is_rhel_like:
-                cmd = f"rpm --query --queryformat '%{{VERSION}}' {self.scylla_pkg()}"
-            else:
-                cmd = f"dpkg-query --show --showformat '${{Version}}' {self.scylla_pkg()}"
-            result = self.remoter.run(cmd, ignore_status=True)
+            return result.stdout.strip()
+        self.log.debug("Unable to get ScyllaDB version using `%s':\n%s\n%s",
+                       result.command, result.stdout, result.stderr)
+        return None
```

Comment:
"None" is default return value, do we really need to have it be explicit?

Reply:
AFAIK, pylint curse if you don't do that

Reply:
Yep. It's because of [R1710 (inconsistent-return-statements)](https://pycodequ.al/docs/pylint-messages/r1710-inconsistent-return-statements.html)